inc/namespace.php

```diff
@@ -384,9 +393,12 @@ function add_fallback_sizes( array $metadata, int $attachment_id ) : array {
 		$metadata['sizes'] = [];
 	}
 
+	$source_url = get_post_meta( $attachment_id, '_amf_source_url', true )
```

Comment:
I suggest to encapsulate getting the source url that future developers don't have to know  to check for a fallback. They can just pass the fallback as a function parameter to something like `get_source_url($attachment_id, $fallback_url)`.

Especially as this is done in multiple places and the next developer might make the same bug.